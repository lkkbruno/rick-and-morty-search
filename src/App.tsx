import React from 'react';
import FindByCharacterPage from './findByCharacter.page';
import { RickAndMortyProvider } from './contexts/rickAndMorty';

const App: React.FC = () => {

  return (
    <RickAndMortyProvider>
      <FindByCharacterPage />
    </RickAndMortyProvider>
  );
};

export default App;
