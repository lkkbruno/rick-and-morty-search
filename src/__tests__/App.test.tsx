import React from 'react';
import { render } from '@testing-library/react';
import App from '../App';

describe('CharacterList', () => {
  it('should match snapshot', async () => {
    const { container } = render(
      <App />
    );

    expect(container).toMatchSnapshot();
  });
});
