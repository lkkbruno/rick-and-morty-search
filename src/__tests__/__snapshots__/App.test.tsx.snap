// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterList should match snapshot 1`] = `
<div>
  <div>
    <header
      class="sc-kAyceB jNtZQo"
    >
      <img
        alt="Rick and Morty"
        class="sc-imWYAI jivJFB"
        src=""
      />
      <input
        class="sc-jXbUNg gwzMty"
        data-testid="search-input"
        placeholder="Search characters by name"
        type="text"
        value=""
      />
    </header>
    <ul
      class="sc-gsFSXq eQVcDd"
    />
  </div>
</div>
`;

exports[`CharacterList should render a list of characters 1`] = `
<div>
  <div>
    <header
      class="sc-kAyceB jNtZQo"
    >
      <img
        alt="Rick and Morty"
        class="sc-imWYAI jivJFB"
        src=""
      />
      <input
        class="sc-jXbUNg gwzMty"
        data-testid="search-input"
        placeholder="Search characters by name"
        type="text"
        value=""
      />
    </header>
    <ul
      class="sc-gsFSXq eQVcDd"
    />
  </div>
</div>
`;
